MinecraftClient/Mapping/CubeFromWorld.cs

```diff
@@ -0,0 +1,184 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace MinecraftClient.Mapping
+{
+    /// <summary>
+    /// A row of blocks that will be mined
+    /// </summary>
+    public class Row
+    {
+        private List<Location> blocksInRow;
```

Comment:
As the variable here will not be changed once created, you may use `readonly` here and remove the public getter.
```suggestion
        public readonly List<Location> BlocksInRow;
```